### PR TITLE
Add a condition for WFL local development.

### PR DESCRIPTION
**Pull request by rexwangcc**
### Purpose

- Add a condition for WFL UI local development according to http://httpd.apache.org/docs/2.4/mod/core.html#if

### Changes

- We could turn off HTTPS redirect for local development.

### Review Instructions

- No instructions.

### CODE REVIEW

**Comment by samanehsan** [comment]
Is this for running WFL locally using minikube?

**Comment by rexwangcc** [comment]
> Is this for running WFL locally using minikube?

Yup! Currently without having the ability to turn off HTTPS locally, it's hard to test the requests to API.

**Comment by rexwangcc** [comment]
@tbl3rd It's still not working yet, it seems only turning the redirect is not enough, we also have to turn the liveness probe and readiness probe off :(

File: ops/cli.sh [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+        fi
+    done
+    test "$ok"
+}
+
+# Wait at least 10 seconds for a result from "@".
+#
+wait_for_result () {
+    local n result
+    for n in 0 1 2 3 4 5 6 7 8 9
```
Comment (ehigham) [inline ops/cli.sh:61]: `for n in {0..9}`

File: helm/wfl/templates/configmap.yaml [hunk 2]
```diff
@@ -6,6 +6,165 @@ metadata:
   labels:
     {{- include "wfl.labels" . | nindent 4 }}
 data:
+  override: |-
```
Comment (ehigham) [inline helm/wfl/templates/configmap.yaml:9]: Is there no other way of doing this? Like a .phony "build" target?

File: helm/wfl/templates/configmap.yaml [hunk 3]
```diff
@@ -6,6 +6,165 @@ metadata:
...
+    #!/bin/sh
+
+    # I'm an override file.  Put things in here to override the openidc-baseimage
+    # defaults or add more settings of your own.
+    #
+    # Note: This script will be executed *after* the main `run.sh` script, but
+    # *before* Apache itself is run.
+
+    # set ALLOW_HEADERS3
```
[the remark this answers is not in the document]
Comment (rexwangcc) [reply helm/wfl/templates/configmap.yaml:18]: Thanks for the comment! This is copied from https://github.com/broadinstitute/openidc-proxy/blob/master/override.sh, I'm still not sure how to make Apache respect the Env Vars yet :(

**Comment by rexwangcc** [comment]
Based on an offline discussion with @tbl3rd , it doesn't make sense to stick with local development. Closing this now!

File: helm/wfl/templates/configmap.yaml (hunk 2 above)
Comment (tbl3rd) [reply helm/wfl/templates/configmap.yaml:9]: Ah … make.

**Pull request closed without merging**